app/jobs/start_sync_job.rb

```diff
@@ -1,12 +1,20 @@
 class StartSyncJob < ApplicationJob
   attr_accessor :sync, :part_size
 
-  def perform(sync_id, part_size = OrganizationExportJob::PART_SIZE)
+  PART_SIZE = OrganizationExportJob::PART_SIZE
+
+  def self.force_sync
+    sync = Sync.create state: SyncMicroMachine::UNSTARTED
```

Comment:
Oops, I forgot that this should actually start the sync in the `SyncMicroMachine::STARTING` state. My bad.